dbt_copilot_helper/COMMANDS.md

```diff
@@ -656,23 +656,20 @@ copilot-helper pipeline generate-config
 
 ## Commands
 
-- [`generate-config` ↪](#copilot-helper-pipeline-generate-config)
+- [`generate` ↪](#copilot-helper-pipeline-generate)
 
-# copilot-helper pipeline generate-config
+# copilot-helper pipeline generate
```

Comment:
I am only noticing this now, but is this missing a description/help text?

Reply:
Added in latest commit 👍